client/my-sites/checkout/composite-checkout/components/checkout-main.tsx

```diff
@@ -683,6 +684,7 @@ export default function CheckoutMain( {
 			<QueryJetpackSaleCoupon />
 			<QuerySitePlans siteId={ updatedSiteId } />
 			<QuerySitePurchases siteId={ updatedSiteId } />
+			<QueryUserPurchases />
```

Comment:
Is there a way to query this conditionally? Perhaps only when the cart key is `no-site`?